scripts/spack/packages/axom/package.py

```diff
@@ -471,17 +474,14 @@ def initconfig_package_entries(self):
             # Grab common devtools root and strip the trailing slash
             path1 = os.path.realpath(spec["cppcheck"].prefix)
             path2 = os.path.realpath(spec["doxygen"].prefix)
-            devtools_root = os.path.commonprefix([path1, path2])[:-1]
-            path_replacements[devtools_root] = "${DEVTOOLS_ROOT}"
-            entries.append("# Root directory for generated developer tools\n")
-            entries.append(cmake_cache_path("DEVTOOLS_ROOT", devtools_root))
+            self.find_path_replacement(path1, path2, path_replacements, "DEVTOOLS_ROOT", entries)
 
-        if "+devtools" in spec and "toss_4" not in self._get_sys_type(spec):
+        if "+devtools" in spec and spec.satisfies("^llvm"):
             # Only turn on clangformat support if devtools is on and not TOSS4
```

Comment:
We're not checking "toss_4" anymore.  Is this comment outdated?

Reply:
This was originally from the machine that was pre-tioga (who's name escapes me). I'll fix the comment.